en/getting_started/sensor_selection.md

```diff
@@ -40,15 +40,6 @@ They are so important because the autopilot does not have other means to detect
 
 For more information and recommended hardware see: [Airspeed Sensors](../sensor/airspeed.md).
 
-## Tachometer

```

Comment:
Same comment like here: https://github.com/PX4/PX4-user_guide/pull/2753#discussion_r1335013837

Reply:
But do we need to keep them in the sensor overview? I didn't remove the tachometer docs, just from the overview (I don't find them important enough for this list, especially as we nowhere in PX4 make use of the RPM feedback). If we want to cover all sensors that have a driver in PX4 then I find that we need to include others first (external IMU, baros, temperature sensors, current sensors etc).

Reply:
I agree "more" with @sfuhrer . I think we should have a list at the end of other peripherals that might be of interest, or even just a link and statement they exist. Let me think.